src/main/java/frameworksdrivers/Gateways/CourseDbGateway.java

```diff
@@ -133,4 +135,30 @@ public void findCourse(EvaluatorDbResponseModel requestModel) {
             e.printStackTrace();
         }
     }
+
```

Comment:
This method doesn't need a change as far as I know, however as we haven't changed how we changed  what we save to sessionstudent table there will be a bug slightly in course enrollment but we can fix this for our final implementation.